src/com/mediaengine/rpc/rpc_cdm_mediaengine_handler.cc

```diff
@@ -15,14 +15,91 @@
  */
 
 #include <fstream>
+#ifdef WPE
+#include "rpc_cdm_mediaengine_handler.h"
+#include <cdm_logging.h>
+#include <opencdm_xdr.h>
+#else	//chrome
 #include "media/cdm/ppapi/external_open_cdm/src/com/mediaengine/rpc/rpc_cdm_mediaengine_handler.h"
 #include "media/cdm/ppapi/cdm_logging.h"
 #include "media/cdm/ppapi/external_open_cdm/src/com/common/rpc/opencdm_xdr.h"
-
+#endif
 namespace media {
 
 std::string rpcServer = "localhost";
+#ifdef WPE
+RpcCdmMediaengineHandler::RpcCdmMediaengineHandler(char *session_id_val,
+                                                   uint32_t session_id_len,
+                                                   uint8_t *auth_data_val,
+                                                   uint32_t auth_data_len) {
+
+  CDM_DLOG() << "RpcCdmMediaengineHandler::RpcCdmMediaengineHandler";
```

Comment:
Here you are un-doing the work by Zoltan in commit for the WPE browser support.

Author: Zoltan Kuscsik <zoltan.kuscsik@linaro.org>  2015-10-16 14:54:18
Committer: Zoltan Kuscsik <zoltan.kuscsik@linaro.org>  2015-10-16 14:54:18
Parent: 02675213391e70305699ace63ebdc0e49fb46047 (Chromium: no symbolic links for OCDM folder)
Branches: chromium-53.0.2785.143, remotes/origin/chromium-53.0.2785.143, remotes/origin/master
Follows: 
Precedes: 

    Make RPC handler a singleton
    
    This is the first step to towards supporting
    multiple CDM session. The RPC handler needs
    to be a singleton.

Which introduced the RpcCdmMediaengineHandler::CreateMediaEngineSession API.

The right approach here I think will be for WPE to also make use of the CreateMediaEngineSession API as well. 

As a general comment, where there are differences between Linaro OpenCDM and WPEOpenCDM (such as this). It not maintainable or long term sustainable to add a #ifdef WPE and create just another code path to maintain. Either WPE support or the Linaro OpenCDM needs to be updated to use the new API, so that one common code path works for both browsers.